plugins/woocommerce-admin/client/customize-store/assembler-hub/sidebar/global-styles/font-pairing-variations/constants.ts

```diff
@@ -558,14 +523,14 @@ export const FONT_PAIRINGS_WHEN_USER_DID_NOT_ALLOW_TRACKING = [
 			elements: {
 				heading: {
 					typography: {
-						fontFamily: 'var(--wp--preset--font-family--inter)',
+						fontFamily: 'var(--wp--preset--font-family--body)',
```

Comment:
We have to use the variable defined by the theme: https://github.com/WordPress/twentytwentyfour/blob/trunk/theme.json#L215